tests/conftest.py

```diff
@@ -436,13 +444,18 @@ def application_docker_container(
     output_directory: Path,
     application_docker_mounts: List[docker.types.Mount],
     application_docker_capabilities: List[str],
+    application_docker_command: Optional[List[str]],
 ) -> Iterable[Container]:
     if not in_container:
         yield None
         return
     else:
         with _application_docker_container(
-            docker_client, application_docker_image, application_docker_mounts, application_docker_capabilities
```

Comment:
Time to use kwargs here, I think

Reply:
Keyword-only? Yeah, I agree.
Yet I would leave `docker_client`, `docker_image` as positional, as they will always be mandatory.